process_blocking.go

```diff
@@ -304,19 +304,19 @@ func (p *blockingProcess) Signal(ctx context.Context, sig syscall.Signal) error
 		case res := <-out:
 			return res
 		case <-ctx.Done():
-			return errors.New("context canceled")
+			return errors.Wrap(ctx.Err(), "context canceled")
```

Comment:
This probably can just return `ctx.Err()`

Reply:
Same as above. I wrapped it with a better message.